rmatch/pom.xml

```diff
@@ -26,6 +26,11 @@
             <version>26.0.2-1</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.ahocorasick</groupId>
+            <artifactId>ahocorasick</artifactId>
+            <version>0.6.3</version>
```

Comment:
Put the version of the ahocorasic thingy into the preamble of the pom file so that we can see all the constants at a single location.

Reply:
Actually. Put it in the parent pom and inherit it downwards.